yarn.lock

```diff
@@ -3382,7 +3382,8 @@ algoliasearch-helper@^3.4.4:
   dependencies:
     events "^1.1.1"
 
-"algoliasearch-v3@npm:algoliasearch@3.35.1", algoliasearch@^3.35.1:
+"algoliasearch-v3@npm:algoliasearch@3.35.1":
+  name algoliasearch-v3
```

Comment:
hmm, this change still will make places fail, I think yarn of a certain version always tries to break the lock file to have only one entry for ones that don't share a name

Reply:
Yeah I reverted that part. It's gonna keep bugging us until we figure out which version of yarn won't do this, or another way to avoid this.